core/src/main/java/org/apache/iceberg/ManifestFiles.java

```diff
@@ -124,13 +124,13 @@ private ManifestFiles() {
   /**
    * Returns a new {@link ManifestReader} for a {@link ManifestFile}.
    *
-   * @param manifest a {@link ManifestFile}
-   * @param io a {@link FileIO}
+   * @param manifest  a {@link ManifestFile}
+   * @param io        a {@link FileIO}
    * @param specsById a Map from spec ID to partition spec
    * @return a {@link ManifestReader}
    */
   public static ManifestReader<DeleteFile> readDeleteManifest(ManifestFile manifest, FileIO io,
-                                                              Map<Integer, PartitionSpec> specsById) {
+      Map<Integer, PartitionSpec> specsById) {
```

Comment:
Nit:   The indent should not be the  correct iceberg-style.  In iceberg,  we usually use the indent like: 

```java
  public static ManifestReader<DeleteFile> readDeleteManifest(ManifestFile manifest, FileIO io,
                                                              Map<Integer, PartitionSpec> specsById) {
```